Project/backend/codebase/graph_creator/gemini.py

```diff
@@ -0,0 +1,125 @@
+import os
+import json
+from datetime import datetime
+from dotenv import load_dotenv
+import google.generativeai as genai
+
+def configure_genai():
+    """
+    Ensure the API key is set in the environment (set it in the .env file or Linux/Mac: export GOOGLE_API_KEY="Your_API_KEY")
+    """
+    load_dotenv("Project/backend/.env", override=True)
```

Comment:
This line needs to be removed since docker already loads the `.env` variables. 
Please update also the `.env.example` with the new needed variables